connection.go

```diff
@@ -341,7 +341,7 @@ func (c *Connection) connect(ctx context.Context) error {
 	// connect
 	transport, err := c.transport.Dial(ctx)
 	if err != nil {
-		c.log.Warning("Connection: error dialing transport: %#v", err)
+		c.log.Debug("Connection: error dialing transport: %#v", err)
```

Comment:
This should be a `%s`, and not a `%#v`.

I'm not convinced this should be a `debug`